Source/WebInspectorUI/UserInterface/Views/DOMTreeElement.js

```diff
@@ -1271,30 +1277,57 @@ WI.DOMTreeElement = class DOMTreeElement extends WI.TreeElement
             return;
         }
 
-        var treeOutline = this.treeOutline;
-        var wasExpanded = this.expanded;
+        let treeOutline = this.treeOutline;
+        let wasExpanded = this.expanded;
 
-        function changeTagNameCallback(error, nodeId)
-        {
+        let newTagName = newText;
+        let attributes = "";
+        let whitespaceIndex = newText.search(/\s/);
+        if (whitespaceIndex !== -1) {
+            newTagName = newText.slice(0, whitespaceIndex);
+            attributes = newText.slice(whitespaceIndex + 1).trim();
```

Comment:
NIT: i think we actually could just use `trimEnd()` because this will either be entirely whitespace (in which case `trim()` and `trimEnd()` will do the same thing) or it'll have content (and even if there is whitespace at the beginning it'll get handled for us in the backend).  not a huge deal tho as i am also a fan of trying to give a slightly nicer value to the protocol so we're sending less data across IPC/etc. :)

Reply:
Turns out the backend isn't that resilient. If we pass it a string with extra space around each attribute it will return an `invalidCharacterError` from [InspectorDOMAgent::setAttributesAsText()](https://github.com/WebKit/WebKit/blob/main/Source/WebCore/inspector/agents/InspectorDOMAgent.cpp#L783).

Likely from this iteration:

https://github.com/WebKit/WebKit/blob/26796239d990f7b51f4dc9418a764feb900d469b/Source/WebCore/inspector/agents/InspectorDOMAgent.cpp#L808-L814

 (Waiting on a build with logs to confirm). 
 The first attribute seems to be applied. See video:
 

https://github.com/user-attachments/assets/8bdb9246-b95f-479b-ad81-69c05317f6cb

I replaced `.trim()` with `.replace(/\s+/g, " ")` in the latest patch to address this.